grass/variables/defaults/roads_speed_bic_defaults

```diff
@@ -0,0 +1,9 @@
+Motorways: 0.0001
```

Comment:
Why not just "0"? You use `parseInt` anyway.

Reply:
Because around line 186 `time_map.js` there is a **avg_speed>0** condition: 
```
grass(this.mapset, `v.extract -r input=m1a_highways_points_connected@${this.mapset} where="avg_speed>0" output=m1a_temp_connections --overwrite`)
```  
When I set it to **avg_speed>=0**, the result is like the picture below - this is in the case of automobile, same thing happen for walking and bicycling as well. So I used **0.0001** instead.


![Screenshot from 2021-02-25 12-50-11](https://user-images.githubusercontent.com/31316150/109149683-31c17c00-7768-11eb-9ed4-549d00733235.png)